website/src/css/custom.less

```diff
@@ -844,4 +838,70 @@ article a:not([data-quicklook-from]):not(.footnote-ref) {
   line-height: 1.5;
   padding-top: 0;
   padding-bottom: 0;
-}
\ No newline at end of file
+}
+
+
+// #region consolidated-sidebar-prototype
```

Comment:
new sidebar design implemented here, intent is to clean this up when we migrate from LESS -> SCSS